peregrine/defaults.py

```diff
@@ -1,4 +1,5 @@
 # Copyright (C) 2014 Swift Navigation Inc.
```

Comment:
This file duplicates functionality in `initSettings.py` and in the command line arguments you have in `peregrine/analysis/tracking_loop.py`. What are your thoughts about standardizing on one method of denoting the metadata of the sample file you're processing?